### DISCUSSION TITLE
Pivot table type function

### DISCUSSION THREAD

**Discussion by FlaminWrap** [Budibase/budibase#5499] [Help] [answered]
Hi All,

Is there any pivot table style function is Budibase. What I am looking for is a way for a data set to be grouped into months and dynamically add each month as it comes. e.g. a row added for May when applications are submitted in May. I have managed to manually do it but it requires a lot of work for every month. Every row value needs its on view and every row needs to be made manually.

![image](https://user-images.githubusercontent.com/97764630/164220059-0dfed6f0-34d5-4871-b508-70e0368aba84.png)

Thank you

**Comment by melohagan** [comment, marked as answer]
Hi @FlaminWrap 

There is no pivot table as such, but you can aggregate data using formulas and views:

- https://docs.budibase.com/docs/formula
- https://docs.budibase.com/docs/views

**Reply by FlaminWrap** [reply]
Thank you for the infomation @melohagan,

I have created a view and tried to group by the CreatedOn field, but it just returns individual timestamps. I created a static formula to extract the date, but there is no option to group by it.

Would it be possible to add a grouping option for formula columns?

**Reply by melohagan** [reply]
Hey @FlaminWrap There's an issue related to that; seems it isn't a possibility, but there are some suggestions in there: https://github.com/Budibase/budibase/issues/2209

**Reply by FlaminWrap** [reply]
Thank you @melohagan, I thought as much. I suppose I could also try the API or using an Automation to feed the required data into another table.
